backend/entities/Role.cs

```diff
@@ -9,6 +9,11 @@ namespace Pims.Dal.Entities
     public class Role : LookupEntity<Guid>
     {
         #region Properties
+        /// <summary>
+        /// get/set - The GUID that identifies this Group in Keycloak.
+        /// </summary>
+        public Guid? KeycloakGroupId { get; set; }
```

Comment:
Using a subordinate property to link with keycloak to free us from some of the issues that integration with keycloak was causing.